lisa/base_tools/mv.py

```diff
@@ -14,13 +15,14 @@ def can_install(self) -> bool:
         return False
 
     def move(
-        self, src_path: str, dest_path: str, overwrite: bool = False, sudo: bool = False
+        self, src_path: str, dest_path: str, overwrite: bool = False,
+        sudo: bool = False, expected_exit_code: Union[None, int] = 0
```

Comment:
add a parameter `ignore_error`, if it's true, set the `expected_exit_code` to `None` in the method. It's easier to understand than exposing `expected_exit_code` directly.